lib/proxy-agent.js

```diff
@@ -42,7 +62,7 @@ class ProxyAgent extends DispatcherBase {
 
     const connect = buildConnector({ ...opts.proxyTls })
     this[kConnectEndpoint] = buildConnector({ ...opts.requestTls })
-    this[kClient] = new Client({ origin: opts.origin, connect })
+    this[kClient] = new Agent({ origin: opts.origin, connect })
```

Comment:
Why this change?

Reply:
```
const Client = require('./agent')
const Agent = require('./agent')
```

Naming was confusing, they were exactly same thing. Am I missing anything?

Reply:
Seems more like the import was incorrect?

Reply:
ok, I'll fix it then

Reply:
done